src/pip/_internal/operations/prepare.py

```diff
@@ -98,17 +100,16 @@ def __init__(
     @property
     def _download_should_save(self):
         # type: () -> bool
-        # TODO: Modify to reduce indentation needed
-        if self.download_dir:
-            self.download_dir = expanduser(self.download_dir)
-            if os.path.exists(self.download_dir):
-                return True
-            else:
-                logger.critical('Could not find download directory')
-                raise InstallationError(
-                    "Could not find or access download directory '%s'"
-                    % display_path(self.download_dir))
-        return False
+        if not self.download_dir:
+            return False
+
+        if os.path.exists(self.download_dir):
+            return True
+
+        logger.critical('Could not find download directory')
+        raise InstallationError(
+            "Could not find or access download directory '%s'"
+            % display_path(self.download_dir))
```

Comment:
Change this to `.format` style?

Reply:
Or rather, let's make that a good first issue, specifically geared toward changing this specific call to `.format`. Then, we can on board someone who can help by modernizing these calls. 🙃

Reply:
@chrahunt could you pretty please file an issue for the above? (listing instructions for this specific change while stating that we'd want to separately improve call sites in the future and that logging is a special case.

Reply:
Sure, will do after we merge this, so I can link to the right line.

Reply:
Will do once this is merged, so I can link to it properly. :+1:

Reply:
Thanks a lot!

Reply:
See #6973.